graphql-dxm-provider/src/main/java/org/jahia/modules/graphql/provider/dxm/node/NodeQueryExtensions.java

```diff
@@ -56,21 +58,29 @@ private Workspace(String workspace) {
         public String getValue() {
             return workspace;
         }
+
+        public static class DefaultWorkspaceSupplier implements Supplier<Object> {
+
+            @Override
+            public Workspace get() {
+                return EDIT;
+            }
+        }
```

Comment:
I wonder if we could directly implement Supplier<Object> at the Workspace class itself, would it be cleaner ? or would it be strange ?

```
@GraphQLDescription("JCR workspace to use for the operations")
    public enum Workspace implements Supplier<Object> {
        
        //..

        @Override
        public Object get() {
            return EDIT;
        }
    }
```

I let you decide :D